tests_tf/test_utils_tf.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+import unittest
+import tensorflow as tf
+
+from cleverhans import utils_tf
+
+
+def numpy_kl_with_logits(q_logits, p_logits):
```

Comment:
This function is already defined in tests_tf/test_utils.py, with the same mistakes. Maybe we should put one version of it in a shared location, and write some tests for it, using extreme values of logits that would break the current version without the numerical stabilization tricks.